v22.1/map-sql-activity-to-app.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Map SQL Activity using an Application Name
+summary: How to set an application name and filter SQL activity associated with the application.
+toc: true
+docs_area: develop
+---
+
+Many relational database management systems enable you to programmatically create and tag your SQL connections with a custom name or label. This practice lets you to quickly identify which part of your application is potentially degraded due to database health and design. By identifying problematic transactions and statements, you can quickly trace back to parts of your applications that generated those transactions and statements for a holistic understanding of the application performance impact as well as potential mitigations and optimization opportunities.
```

Comment:
Suggest changing "many DBMSes" to something more concrete like "CockroachDB enables you to programmatically create and tag ..." 

Suggest para break at "By identifying ..." (this is a long para)

Suggest linking "optimization opportunities" to https://deploy-preview-13558--cockroachdb-docs.netlify.app/docs/v22.1/make-queries-fast.html

Reply:
Suggest adding another small paragraph that says something like "on this page you will find instructions to do X" or something similar, to let the reader know what info is on this page so they can decide whether to keep reading